calendar/calendar-impl/impl/src/java/org/sakaiproject/calendar/impl/BaseCalendarService.java

```diff
@@ -142,6 +142,8 @@
 	
 	private Cache cache = null;
 	
+	private String m_calRefList = "";
```

Comment:
There is only on instance of the `BaseCalendarService` when Sakai is running, however multiple threads may call `handleAccessIcalCommon` concurrently so they will all be fighting over the instance variable `m_calRefList` changes get logs.